collated/main/limcel.md

```diff
@@ -0,0 +1,928 @@
+        try {
+            model.addSchedule(schedule);
+        } catch (PersonNotFoundException e) {
+            return new CommandResult(PERSON_NOT_FOUND);
+        }
+
+        EventsCenter.getInstance().post(new JumpToListRequestEvent(targetIndex));
+        return new CommandResult("Added " + personAddedToSchedule.getName().toString()
+                + " to consultations schedule " + "on " + schedule.getDate().toString() + ".\n"
+                + "Use 'viewsch' or 'viewschedule' command to view all your schedules.");
+
+    }
+
+    @Override
+    public boolean equals(Object other) {
+        return other == this // short circuit if same object
+                || (other instanceof ScheduleCommand // instanceof handles nulls
+                && this.date.getTimeInMillis() == ((ScheduleCommand) other).date.getTimeInMillis())
+                && this.targetIndex.getZeroBased() == ((ScheduleCommand) other).targetIndex.getZeroBased();
+    }
+
+    /**
+     * Returns true if appointment date set to after current time
+     */
+    private boolean isDateValid() {
+        requireNonNull(date);
+        Calendar calendar = Calendar.getInstance();
+        return !date.getTime().before(calendar.getTime());
+    }
+}
+```
+###### \java\seedu\address\logic\commands\SortCommand.java
+``` java
+/**
+ * Sorts all contacts in alphabetical order by their names from the address book.
+ * Command is case-insensitive
+ */
+public class SortCommand extends Command {
+    public static final String COMMAND_WORD = "sort";
+    public static final String COMMAND_ALIAS = "st";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + " OR "
+            + COMMAND_ALIAS
+            + ": Sort all contacts names in alphabetical order. ";
+
+    public static final String MESSAGE_SUCCESS = "All contacts are sorted alphabetically by name.";
+
+    private ArrayList<ReadOnlyPerson> contactList;
+
+    public SortCommand() {
+        contactList = new ArrayList<>();
+    }
+
+    @Override
+    public CommandResult execute() {
+        model.updateFilteredPersonList(PREDICATE_SHOW_ALL_PERSONS);
+        model.sortByPersonName();
+        return new CommandResult(MESSAGE_SUCCESS);
+    }
+}
+```
+###### \java\seedu\address\logic\commands\TabCommand.java
+``` java
+public class TabCommand extends Command {
+    public static final String COMMAND_WORD = "tab";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + " : Parameters: INDEX (must be a positive integer) "
+            + "Example: " + COMMAND_WORD + " 1 ";
+
+    public static final String MESSAGE_SELECT_TAB_SUCCESS = "Selected Tab: %1$s";
+    public static final String MESSAGE_INVALID_TAB_INDEX = "Invalid Tab Value";
+
+    // there are only 2 types of graphs available for display (Graph / Bar)
+    public static final int NUM_TAB = 2;
+
+    private final Index targetIndex;
+
+    /**
+     * @param targetIndex of the TabPane in the graph panel for switching
```

Comment:
The start of the header comment does not follow the convention (i.e. first sentence should be something like `Selects next line …` instead of `Select next line` or `This method is for selecting …`)